.travis.yml

```diff
@@ -163,7 +163,7 @@ script:
   - test.pyomo -v --cat=$CATEGORY pyomo `pwd`/pyomo-model-libraries
 
   # Run documentation tests
-  - cd doc/OnlineDocs; make doctests -d; cd ../..
+  - cd doc/OnlineDocs && make doctest -d && cd ../..
```

Comment:
This worries me.  If doctest fails, then the CWD is not restored back to outside of the doc directory.  I have frequently used:
```
pushd doc/OnlineDocs
make doctest -d # possible also '|| exit 1'
popd
```
when I was working with scripts that had to change/restore the CWD.